indico/modules/events/payment/blueprint.py

```diff
@@ -24,6 +25,8 @@
                  'event_settings_edit', RHPaymentSettingsEdit, methods=('GET', 'POST'))
 _bp.add_url_rule('/event/<int:event_id>/manage/payments/method/<method>',
                  'event_plugin_edit', RHPaymentPluginEdit, methods=('GET', 'POST'))
+_bp.add_url_rule('/event/<int:event_id>/manage/registration/<int:reg_form_id>/registrations/change_price',
```

Comment:
```suggestion
_bp.add_url_rule('/event/<int:event_id>/manage/registration/<int:reg_form_id>/registrations/change-price',
```

underscores in urls are ugly, we don't use them anywhere else